config/confighttp/compression.go

```diff
@@ -58,17 +58,10 @@ var availableDecoders = map[string]func(body io.ReadCloser) (io.ReadCloser, erro
 		}
 		return zr, nil
 	},
+	//nolint:unparam // ignoring lint since method needs to fit typedef
```

Comment:
lost about this comment, can you extend?

Reply:
Can do, this was golangci-lint calling out that this method should exclude the `error` being returned as a the function definition since it will always be nil.

Reply:
Thank you.